partitura/utils/music.py

```diff
@@ -471,6 +193,88 @@ def _transpose_note(note, interval):
         )
 
 
+def transpose_note_old(step, alter, interval):
```

Comment:
is this function still useful? Or can it be deleted?

Reply:
It is not used anywhere else. I would like to keep it for now and delete it on the next release to make sure no errors with the new method arrise.